openhands/utils/chunk_localizer.py

```diff
@@ -93,7 +93,7 @@ def get_top_k_chunk_matches(
     ]
     sorted_chunks = sorted(
         chunks_with_lcs,
-        key=lambda x: x.normalized_lcs,  # type: ignore
+        key=lambda x: 0.0 if x.normalized_lcs is None else x.normalized_lcs,
```

Comment:
```suggestion
        key=lambda x: x.normalized_lcs,
```

I believe `normalized_lcs` always returns float?